package.json

```diff
@@ -6,9 +6,13 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^1.2.2",
+    "link": "^1.5.1",
```

Comment:
You shouldn't need this package Khaled, the <Link> component is part of the react-router-dom package